tools/bin/ariaExtract.py

```diff
@@ -142,6 +142,34 @@ def main():
     logging.basicConfig(level=log_level, format=ARIAtools.util.log.FORMAT)
     LOGGER.info('Extract Product Function')
 
+    # Check whether all necessary inputs were specified.
+    # some products require a DEM to extract -- if any of those are requested,
+    # ensure that a valid DEM is specified
+    if args.layers:
```

Comment:
Recommend to use the more pythonic version of this: `args.layers is not None:`

Reply:
@rzinke Just this last comment here, thanks!

Reply:
Done, thanks @alexfore !